bats_test/downloads/file-downloads/publlic_files.bats

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bats
```

Comment:
**TODO** rename this file to `public_files.bats`

Reply:
Fixed